Trustroots/trustroots
# Offer map: resize on window resize

When adding the new base `Map` component in https://github.com/Trustroots/trustroots/pull/1156, I intentionally left window resize-refreshing for a follow-up PR. Here it is. :-)

#### Proposed Changes

* Add window resize event handler to Map base component. Debounce event to once per second to avoid performance issues.

#### Testing Instructions

* Either at "meet" page (after adding one meet offer) or at a profile page (after adding a hosting offer)...
* Observe how zooming and panning works
* Drag browser window to different size so that map container size changes and observe map canvas accommodate new width
* Zoom/pan around, then resize again and observe zoom and location be preserved

Before the change:
![image](https://user-images.githubusercontent.com/87168/77255733-4cdf5e00-6c72-11ea-9df4-f0dd68af7823.png)

After the change:
![image](https://user-images.githubusercontent.com/87168/77255739-55379900-6c72-11ea-9858-82a9c737af04.png)
